content/news/XXXX-XX-XX-jack-zero-latency.md

```diff
@@ -0,0 +1,43 @@
+title: "Has JACK zero latency?"
+authors: Daniel Schürmann
+tags: 2.3, jack
+date: 2021.03.18 02:42:18
+
+In regular intervals, we discuss how much latency the JACK layer introduces. Here we can read that Jack adds zero latency: [JACK FAQ](https://jackaudio.org/faq/no_extra_latency.html)
+This is true on its own, because Jack uses directly the buffer provided by ALSA to mix the audio sources together. ALSA has a second buffer that is used to feed the samples into the hardware. That's all.
+
+However in the case of Mixxx another buffer is used, because Mixxx has its own real time mixing stage and uses the portaudio abstraction layer.
```

Comment:
```suggestion
However in the case of Mixxx another buffer is used, because Mixxx has its own realtime mixing stage and uses the PortAudio abstraction layer.
```